torchci/lib/bot/mergeBot.ts

```diff
@@ -4,42 +4,44 @@ function mergeBot(app: Probot): void {
   const mergeCmdPat = new RegExp("@pytorch(merge|)bot\\s+merge\\s+this");
   const revertCmdPat = new RegExp("@pytorch(merge|)bot\\s+revert\\s+this");
   app.on("issue_comment.created", async (ctx) => {
-    const commentBody = ctx.payload.comment.body;
-    const owner = ctx.payload.repository.owner.login;
-    const repo = ctx.payload.repository.name;
-    const commentId = ctx.payload.comment.id;
-    const prNum = ctx.payload.issue.number;
-    if (commentBody.match(mergeCmdPat)) {
-      await ctx.octokit.repos.createDispatchEvent({
-        owner,
-        repo,
-        event_type: "try-merge",
-        client_payload: {
-          pr_num: prNum,
-        },
-      });
-      await ctx.octokit.reactions.createForIssueComment({
-        comment_id: commentId,
-        content: "+1",
-        owner,
-        repo,
-      });
-    }
-    if (commentBody.match(revertCmdPat)) {
-      await ctx.octokit.repos.createDispatchEvent({
-        owner,
-        repo,
-        event_type: "try-revert",
-        client_payload: {
-          pr_num: prNum,
-        },
-      });
-      await ctx.octokit.reactions.createForIssueComment({
-        comment_id: commentId,
-        content: "+1",
-        owner,
-        repo,
-      });
+    if (ctx.payload.issue.pull_request) {
```

Comment:
To avoid pyramid of doom, can you simply return early and mark comment as confusing
```suggestion
    if (!ctx.payload.issue.pull_request) {
       await ctx.octokit.reactions.createForIssueComment({
                 comment_id: commentId,
          content: "confused",
          owner,
          repo,
        });
       return;
    }
```